### PR TITLE
fix different attribute order

### PR DESCRIPTION
**Pull request by waldemar-schneider**
fixes the issue as reported in #86

### CODE REVIEW

File: src/main/java/de/neuland/jade4j/lexer/token/Attribute.java [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-import java.util.HashMap;
+import java.util.LinkedHashMap;
 import java.util.Map;
 
 import de.neuland.jade4j.parser.node.ExpressionString;
 
 public class Attribute extends Token {
 
-	Map<String, Object> attributes = new HashMap<String, Object>();
+	Map<String, Object> attributes = new LinkedHashMap<String, Object>();
```
Comment (marlonbernardes) [inline src/main/java/de/neuland/jade4j/lexer/token/Attribute.java]: @waldemar-schneider Seems good! I would only suggest for you to reduce the accessibility of `attributes` by changing it from `Map<String, Object> attributes` to `private Map<String, Object> attributes`.

**Comment by waldemar-schneider** [comment]
@naltatis can you check this pull request?

**Comment by marlonbernardes** [comment]
@waldemar-schneider  This project seems abandoned. One month and no responses...

**Comment by naltatis** [comment]
Sorry for the slow responses. I will definitely look into this in the next says.

**Comment by marlonbernardes** [comment]
@naltatis That's great to hear! :)

**Comment by waldemar-schneider** [comment]
Is there any progress in this PR?

**Comment by waldemar-schneider** [comment]
@marlonbernardes you were right. This project seems abandoned...

**Comment by naltatis** [comment]
This be fixed with the merged PR cae9c91142f3c8e560ffd062d51de6ca51d63427 from @IRus.

**Pull request closed without merging**